pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/documentation-types.md

```diff
@@ -0,0 +1,274 @@
+   </td>
+   <td>Dive deep into project nuances. There may be many ways to accomplish the tutorial, but we are presenting <em>this</em> way.
+   </td>
+  </tr>
+  <tr>
+   <td>Provide a simplified environment and tools to setup. 
+   </td>
+   <td>Assume the developer has familiarity with product and expert level coding ability. 
+   </td>
+  </tr>
+  <tr>
+   <td>Keep sample visually simplistic. 
+   </td>
+   <td>Complicate for style sake, unless the tutorial is about styling. 
+   </td>
+  </tr>
+  <tr>
+   <td>Provide a screenshot of each step and finished demo.
+   </td>
+   <td>Only provide code samples. 
+   </td>
+  </tr>
+  <tr>
+   <td>Create a call to action. Point the developer to what they should follow up with.
+   </td>
+   <td>Conflate the example with further explanation. Consider opening an issue for a guide or tutorial if you feel there isn’t sufficient follow-up.
+   </td>
+  </tr>
+</table>
+
+
+
+## Advanced tutorial
+
+Advanced tutorials help developers accomplish a specific task. It assumes the developer has some familiarity with AMP. It should demonstrate how to build an experience, integrate a feature, or address implementation tasks. 
+
+amp.dev examples:
+
+
+
+*   [How to configure basic analytics for your AMP pages]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/optimize-and-measure/tracking-engagement.md', locale=doc.locale).url.path}}?format=websites)
+*   [Add custom JavaScript to AMP pages with amp-script]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/develop/custom-javascript-tutorial.md', locale=doc.locale).url.path}}?format=websites)
+*   [Turn your AMP site into a PWA]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/optimize-and-measure/amp_to_pwa.md', locale=doc.locale).url.path}}?format=websites)
+
+<table>
+  <tr>
+   <td>
+<strong>Do</strong>
+   </td>
+   <td><strong>Don’t</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Provide step-by-step instructions with a clear end project. 
+   </td>
+   <td>Provide exhaustive details and overelaborate concepts. 
+   </td>
+  </tr>
+  <tr>
+   <td>Provide code samples or downloadable starter code. Additionally, make the final and complete project downloadable.  
+   </td>
+   <td>Provide alternative examples or process to reach the end result.
+   </td>
+  </tr>
+  <tr>
+   <td>Create a plug and play environment. 
+   </td>
+   <td>Link out to a set-up tutorial. Tutorials should be self contained. 
+   </td>
+  </tr>
+</table>
+
+
+
+## Introductory Guide
+
+An introductory guide provides an overview of relevant information to get started with AMP. It should identify the feature, describe what it is, and finish with what it does. Introductory guides will introduce a developer to the basic requirements of the feature without directing them to implement something in a specific way. If you’re walking through a process step by step with code samples, you’re probably writing a tutorial. 
```

Comment:
Maybe also mention the difference between reference docs and and introductory guide?